test/envoye2e/env/var.go

```diff
@@ -0,0 +1,302 @@
+// Copyright 2019 Istio Authors
```

Comment:
Remove this. You can inline any references to the functions here.

Reply:
Removed